### ISSUE TITLE
Experimental setup

### ISSUE THREAD

**Issue by jiangyu-0709** [p-mayo/cauchycsc#1]
Hello, I'm going to quote your article, but there is still a little confusion: how do you set up the experiment? Are the pictures you use to train CSC network and the pictures you use to test the same? Are different images used as test set and training set respectively? I would appreciate it if you could take the time to answer my question.
